### PR TITLE
11.0 mig document url

### PR DESCRIPTION
**Pull request by ernestotejeda**
cc @Tecnativa

### CODE REVIEW

**Comment by rafaelbn** [comment]
cc @Tecnativa

**Review by pedrobaeza — APPROVED** [review]
Working as expected.

File: document_url/__init__.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# © 2014 Serv. Tecnol. Avanzados (http://www.serviciosbaeza.com)
```
Comment (cubells) [inline document_url/__init__.py]: Please remove every char `©` by the word `Copyright` in all files.

File: document_url/readme/USAGE.rst [hunk 2]
```diff
@@ -0,0 +1,4 @@
+To use this module, you need to:
+
+ * go to menu more and click on add an url.
+ * Insert name and url into the wizard.
```
Comment (cubells) [inline document_url/readme/USAGE.rst]: Remove the first blank space in these two lines or list are considered in a quote.

**Review by cubells — APPROVED** [review]
Minor changes

:+1: 

LGTM
Code review and test

**Review by jjscarafia — APPROVED** [review]
:+1: Functional review

**Pull request merged**